### PR TITLE
Format stats for frontend

### PR DESCRIPTION
**Pull request by vbrown608**
* Serve domain count rather than percent for top-million scan
* Serve data to chart as a json array of x-y coordinates, sorted along the x axis

### CODE REVIEW

File: db/sqldb.go [hunk 1]
```diff
@@ -132,7 +132,7 @@ func (db *SQLDatabase) GetMTASTSStats(source string) (stats.Series, error) {
 		if err := rows.Scan(&a.Time, &a.WithMXs, &a.MTASTSTesting, &a.MTASTSEnforce); err != nil {
 			return stats.Series{}, err
 		}
-		series[a.Time.UTC()] = a.PercentMTASTS()
+		series[a.Time.UTC()] = float64(a.TotalMTASTS())
```
Comment (sydneyli) [inline db/sqldb.go]: why does this have to be a float?

File: stats/stats.go [hunk 2]
```diff
@@ -67,6 +70,22 @@ func ImportRegularly(store Store, interval time.Duration) {
 // library prefers.
 type Series map[time.Time]float64
 
+// MarshalJSON marshals a Series to the format expected by chart.js.
```
Comment (sydneyli) [inline stats/stats.go]: Link to chart.js docs? https://www.chartjs.org/docs/latest/

Comment (vbrown608) [reply stats/stats.go]: Sure!

File: db/sqldb.go (hunk 1 above)
Comment (vbrown608) [reply db/sqldb.go]: The underlying type of `Series` (which this fn returns) is `map[time.Time]float64`

Comment (vbrown608) [reply db/sqldb.go]: The underlying type of `Series` (which this fn returns) is `map[time.Time]float64`. That makes it also able to hold a series of percents, as in `GetMTASTSLocalStats`.

This inconsistency gets cleaned up in PR #239, which changes the underlying type of `Series` to `[]checker.AggregatedScan`

Comment (sydneyli) [reply db/sqldb.go]: Ooh okay I see! Thanks.

**Pull request merged**